README.md

```diff
@@ -25,3 +63,12 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+T(n) = log3n + Θ(1)
+
+Θ(log3n)
+
+In each recursive call, the input size is divided into three subproblems, represented by oneThird and twoThird. This results in a recurrence relation of 
+T(n) = log3n + Θ(1).
+
+The work done at each level of the recursion is Θ(log3n), as each element in the array is processed
```

Comment:
...so the function doesn't even look at each element? Then how can it compute the sum?